tools/nl/svindex_differ/README.md

```diff
@@ -6,15 +6,13 @@ runs a bunch of golden queries (in [queryset.csv](queryset.csv)) against a
 
 This should be run, as below, while updating the SV index (following
 instructions [here](../embeddings)) to compare against the version
-currently checked-in:
+currently checked-in. Note that `small` ro `medium` refers to the PROD
```

Comment:
Can we move the index differ change to followon PR, this PR as is, is big enough?

Reply:
Done